sdk/keyvault/azure-security-keyvault-keys/test/key_client_test.cpp

```diff
@@ -18,14 +18,81 @@ TEST(KeyClient, initClient)
   EXPECT_NO_THROW(KeyClient keyClient("vaultUrl", credential));
 }
 
-TEST(KeyClient, DISABLED_SendRequestDefault)
+TEST(KeyClient, DISABLED_GetKey)
 {
-  auto credential
-      = std::make_shared<Azure::Identity::ClientSecretCredential>("tenantID", "AppId", "SecretId");
-  KeyClient keyClient("vaultUrl", credential);
+  auto credential = std::make_shared<Azure::Identity::ClientSecretCredential>(
+      "tenantId", "clientId", "secrectId");
+  KeyClient keyClient("https://vivazqukeyvault.vault.azure.net/", credential);
   auto r = keyClient.GetKey("KeyName");
-  auto t = r.ExtractValue();
-  auto rr = r.ExtractRawResponse();
+  auto keyVaultKey = r.ExtractValue();
+
+  EXPECT_EQ(keyVaultKey.Name(), "KeyName");
+}
+
+TEST(KeyClient, DISABLED_CreateKey)
+{
+  auto credential = std::make_shared<Azure::Identity::ClientSecretCredential>(
+      "tenantId", "clientId", "secrectId");
+  KeyClient keyClient("https://vivazqukeyvault.vault.azure.net/", credential);
+  auto r = keyClient.CreateKey("myKey2", KeyTypeEnum::Ec);
```

Comment:
Note that creating keys this way is an "advanced" way. We have pseudo-overloads for RSA and EC. Will those be added later?

Reply:
Yes, Those overloads would be calling this general createKey API right?

Reply:
That's a way, yes.

Reply:
Yes, my goal is to have the same APIs that .NET currently is offering.

Reply:
Just for clarification, other languages have that as well. We strive for idiomatic consistency. Naming is usually the same, but patterns may different. We do have some naming differences, though: e.g. `listPropertiesOfKeys` in other languages is `GetPropertiesOfKeys` in .NET since we don't use the "list" verb. Not sure about C++ - I've seen plenty of either or no prefix at all, like in STL often times.